impl/src/main/java/io/kroxylicious/testing/kafka/common/KafkaClusterConfig.java

```diff
@@ -126,6 +133,9 @@ public static KafkaClusterConfig fromConstraints(List<Annotation> annotations) {
             if (annotation instanceof ClusterId) {
                 builder.kafkaKraftClusterId(((ClusterId) annotation).value());
             }
+            if (annotation instanceof Version) {
+                builder.kafkaVersion(((Version) annotation).value());
```

Comment:
Can their  & should there be some validation of versions here? 

Do we actually want to even suggest support for Kafka `0.11` for example? 

I'm quite happy to see version validation addressed as a separate PR.